man-roxygen/boilerplate.R

```diff
@@ -0,0 +1,2 @@
+#' @return All tidying methods return a data.frame without rownames, whose
```

Comment:
This boilerplate is also from the old documentation strategy, and I don't want to revert.